packages/atomic/src/components.d.ts

```diff
@@ -2027,7 +2027,7 @@ export namespace Components {
         /**
           * The name of the description field to use.
          */
-        "field": 'ec_description' | 'ec_shortdesc';
+        "field": 'ec_description' | 'ec_shortdesc' | 'excerpt';
```

Comment:
Would it be better to have a distinct component for the `excerpt` (e.g., `atomic-product-excerpt`) since it behaves differently from `ec_description` and `ec_shortdesc`?

Reply:
Im not against it